src/main/java/core/basesyntax/service/FileWriter.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public class FileWriter implements Writer {
+    private static final String PATTERN = "(\\w+,\\w+\\r*\\n*)(\\w+,\\d+\\r*\\n*)*";
+
+    @Override
+    public boolean writeToFile(String report, String filePath) {
+
```

Comment:
same(
please check all your classes, both source and test

we don't need empty lines after method signatures or class declarations